08-communicating-results.Rmd

```diff
@@ -143,41 +147,42 @@ trust_gov_gt %>%
   
```

Comment:
This footnote is long and doesn't warp in PDF.  Started to look into this and I think the solution is here, just need time to try: https://github.com/rstudio/rmarkdown/issues/646